scripts/check_for_docs_deps_changes

```diff
@@ -3,7 +3,7 @@
 CHANGED_FILES=$(git diff HEAD origin/develop --name-only)
 printf '%s\n' "[CHANGED FILES]" "${CHANGED_FILES[@]}" ""
 
-DEPENDENCIES=$(python scripts/trace_docs_deps.py docs)
+DEPENDENCIES=$(python scripts/trace_docs_deps.py)
```

Comment:
We're always running this on docs so I didn't think there was any need to use `sys.argv`